setup.wxs

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
+  <Product Name='Browserpass' Id='ABCDDCBA-86C7-4D14-AEC0-86416A69ABDE' UpgradeCode='ABCDDCBA-7349-453F-94F6-BCB5110BA4FD' Language='1033' Codepage='1252' Version='3.0.6' Manufacturer='Browserpass'>
+
+    <Package Id='*' Keywords='Installer' Manufacturer='Browserpass' InstallerVersion='100' Languages='1033' Compressed='yes' SummaryCodepage='1252' />
+    <Media Id='1' Cabinet='Sample.cab' EmbedCab='yes' DiskPrompt="CD-ROM #1" />
+
+    <Directory Id='TARGETDIR' Name='SourceDir'>
+      <Directory Id='ProgramFilesFolder' Name='PFiles'>
+        <Directory Id='INSTALLDIR' Name='Browserpass'>
+          <Component Id='MainExecutable' Guid='ABCDDCBA-83F1-4F22-985B-FDB3C8ABD471'>
+            <File Id='BrowserpassEXE' Name='browserpass-windows64.exe' Source='browserpass-windows64.exe' KeyPath='yes'/>
+            <File Id='ChromiumHostJSON' Name='chromium-host.json' Source='browser-files/chromium-host.json' />
+            <File Id='ChromiumPolicyJSON' Name='chromium-policy.json' Source='browser-files/chromium-policy.json' />
+            <File Id='FirefoxHostJSON' Name='firefox-host.json' Source='browser-files/firefox-host.json' />
+            <RegistryKey Id='ChromeNativeMessaging' Root='HKCU' Key='Software\Google\Chrome\NativeMessagingHosts\com.github.browserpass.native' >
```

Comment:
Given that we install in a "global" folder, let's also configure browsers in `HKLM` to have effect for all users.

Reply:
actually, I shouldn't have suggested to configure _both_, having only `HKLM` would probably be enough 🤔